inc/class-wpseo-image-utils.php

```diff
@@ -243,7 +243,11 @@ public static function get_file_size( $image ) {
 		// If the file size for the file is over our limit, we're going to go for a smaller version.
 		// @todo Save the filesize to the image metadata.
 		// phpcs:ignore Generic.PHP.NoSilencedErrors.Discouraged -- If file size doesn't properly return, we'll not fail.
-		return @filesize( self::get_absolute_path( $image['path'] ) );
+		if(file_exists(self::get_absolute_path( $image['path'] ))) {
+			return @filesize( self::get_absolute_path( $image['path'] ) );
```

Comment:
This `@` here silences the errors. The comment line in line 245 just above needs to be right above this one to silence the error about it.